backend/project_name/settings/base.py

```diff
@@ -127,6 +128,9 @@ def base_dir_join(*args):
 CELERY_ACKS_LATE = True
 CELERY_TIMEZONE = TIME_ZONE
 
+# Redbeat
+redbeat_redis_url = config("REDBEAT_REDIS_URL", default="")
```

Comment:
does this affect localhost anyway (dev running)?
does this need to be lowercase?

Reply:
It doesn't affect, I think I can move it into the production file.

About the lowercase, yes https://redbeat.readthedocs.io/en/latest/config.html#celery-3-x-config-names (we're using > 4)